src/Materials/effect.ts

```diff
@@ -12,12 +12,17 @@ import { IMatrixLike, IVector2Like, IVector3Like, IVector4Like, IColor3Like, ICo
 import { ThinEngine } from '../Engines/thinEngine';
 import { IEffectFallbacks } from './iEffectFallbacks';
 import { ShaderStore as EngineShaderStore } from '../Engines/shaderStore';
+import { ShaderLanguage } from "./shaderLanguage";
+import { CodeStringParsingTools } from "../Misc/codeStringParsingTools";

```

Comment:
This should probably not be here as it is a pretty large dependency added in our min build ?

Reply:
This code was in `ShaderCodeInliner` previously, which is in the min build I guess?

I think I can move the dependency to `ShaderProcessor` but I don't know if it's better?

Reply:
Code moved to `ShaderProcessor`